### ISSUE TITLE
Show enchantments as list

### ISSUE THREAD

**Issue by codehz** [CutFlame/MCDSaveEdit#26]
Since the game is not limited the count... you could literally using more than 3 or even 9 (e.g. full enchantment)

**Comment by CutFlame** [comment]
That is true, however not possible to achieve in the actual game. I'm trying to shy away from this behavior because it's not guaranteed to be compatible in the future.
